docs/api/permissions.rst

```diff
@@ -0,0 +1,521 @@
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length
+    Content-Length: 288
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 09:48:47 GMT
+    Server: waitress
+
+    {
+        "documentation": "https://kinto.readthedocs.org/",
+        "hello": "cloud storage",
+        "settings": {
+            "cliquet.batch_max_requests": 25
+        },
+        "url": "http://localhost:8888/v1/",
+        "userid": "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6",
+        "version": "1.4.0"
+    }
+
+
+In this case the user id is: ``basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6``
+
+.. note::
+
+    In case of sharing, users may need to register their user id
+    associated with their email address for other user to add permission
+    to them.
+
+
+Object permissions
+==================
+
+On each object, it is possible to add ``write`` and ``read``
+permissions as well as ``sub-object:create`` ones.
+
+.. note::
+
+    In Kinto, object inherit their parents ones. i.e: giving the read
+    permission to a user on a bucket gives read permissions on all
+    objects stored in the bucket.
+
+
+For instance, we can look at our own bucket permissions.
+
+.. code-block::
+
+    $ http GET http://localhost:8888/v1/buckets/default --auth user:pass
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
+    Content-Length: 187
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 09:59:30 GMT
+    Etag: "1437040770742"
+    Last-Modified: Thu, 16 Jul 2015 09:59:30 GMT
+    Server: waitress
+
+    {
+        "data": {
+            "id": "46524be8-0ad7-3ac6-e260-71f8993feffa", 
+            "last_modified": 1437040770742
+        }, 
+        "permissions": {
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+
+We can look at our own tasks collections permissions.
+
+.. code-block::
+
+    $ http GET http://localhost:8888/v1/buckets/default/collections/tasks --auth user:pass
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Content-Length, Last-Modified, ETag
+    Content-Length: 156
+    Content-Type: application/json; charset=UTF-8
+    Date: Thu, 16 Jul 2015 10:00:30 GMT
+    Etag: "1437040830468"
+    Last-Modified: Thu, 16 Jul 2015 10:00:30 GMT
+    Server: waitress
+
+    {
+        "data": {
+            "id": "tasks", 
+            "last_modified": 1437040830468
+        }, 
+        "permissions": {
+            "write": [
+                "basicauth:631c2d625ee5726172cf67c6750de10a3e1a04bcd603bc9ad6d6b196fa8257a6"
+            ]
+        }
+    }
+
+
+Managing object permissions
+===========================
+
+You can use PUT or PATCH to update or replace :term:`ACLs <ACL>` for an object.
```

Comment:
Permissions can be specified during the creation of an object, and can later be updated ...